spec/features/userlist_spec.rb

```diff
@@ -26,3 +33,23 @@
     expect(page).to have_content '@'
   end
 end
+
+feature 'Banning users' do
+  scenario 'Moderator can ban/unban users' do
+    moder = create(:user, role: :moderator,
+                          banned: false)
+    repor = create(:user, role: :reporter,
```

Comment:
@igorkasyanchuk, corrected. I know it's not an excuse, but it was written on the pilot phase and I just forgot to fix it. So now I will always write code as if it will not be changed.